docs/framework/30scoring/implementation.md

```diff
@@ -1,9 +1,51 @@
 ## Implementation Risk Scoring
 
-No aspects of [Implementation Risk](../20categories/30implementation/protocol-implementation-risk.md#protocol-implementation-risk) have been considered when determining the Implementation Risk Score.
+The following aspects of [Implementation Risk](../20categories/30implementation/protocol-implementation-risk.md#protocol-implementation-risk) are currently not taken into account when assessing the Implementation Risk Score: 
+[Mixing of Control and Data Plane](../20categories/30implementation/mixing-control-data-flow.md), 
+[Role Based Access Control](../20categories/30implementation/access-control.md),
+[Upgradability](../20categories/30implementation/upgrade.md),
+[Secret Storage](../20categories/30implementation/secret-storage.md),
+[Product Development Maturity](../20categories/30implementation/maturity.md),
+[Well Known Platform](../20categories/30implementation/known-platform.md),
+[Well Known Smart Contract Programming Language](../20categories/30implementation/known-language.md),
+[Formal Verification](../20categories/30implementation/formal-verification.md),
+[Testing](../20categories/30implementation/testing.md),
+[Open Source Code](../20categories/30implementation/open-source.md),
+[Verified Code on Block Explorer](../20categories/30implementation/verified-code.md),
+[Documentation](../20categories/30implementation/documentation.md),
+[Bug Bounty](../20categories/30implementation/bug-bounty.md),
+[Ability to Pause Project](../20categories/30implementation/pause.md),
+and 
+[Ability to Ban Addresses](../20categories/30implementation/ban-address.md).
+
 
 The equation for the *Implementation Risk Score* is:
 
 ```
-Implementation Risk Score = 10
+Implementation Risk Score = Code Auditing Score
+```
+
+### Code Auditing Score
+Code auditing is described in detail in the [Code Auditing](../20categories/30implementation/audit.md) section. The equation for the *Code Auditing Score* is:
+
+```
+Code Audit Score = (I011 + I012 + I014 + I1015) x I013
 ```
+
+The *Code Auditing Score* ranges from 0 to 10.
```

Comment:
I think it is a bit counter-intuitive to me that a high risk score means lower risk.

Reply:
I see your point. 

I suggest is that this is tackled in a separate PR, as all of the existing scoring assumes larger numbers are good.